src/radical/pilot/agent/lm/spark.py

```diff
@@ -37,10 +37,10 @@ def lrms_config_hook(cls, name, cfg, lrms, logger):
         if not os.environ.get('SPARK_HOME'):
 	        logger.info("Downloading Apache Spark..")
 	        try:    
-		        VERSION = "1.5.2"
-		        subprocess.check_call("wget http://d3kbcqa49mib13.cloudfront.net/spark-1.5.2-bin-hadoop2.6.tgz".split())
-		        subprocess.check_call('tar -xzf spark-1.5.2-bin-hadoop2.6.tgz'.split())
-		        subprocess.check_call(("mv spark-1.5.2-bin-hadoop2.6 spark-" + VERSION).split())
+		        VERSION = "2.0.2"
+                        subprocess.check_call("wget http://d3kbcqa49mib13.cloudfront.net/spark-2.0.2-bin-hadoop2.7.tgz".split())
```

Comment:
I am ssurprised by that, Python is usually picky about indentation changes like this.  Please do  fix it anyway, will you?  Thanks!